src/parser_urdf.cc

```diff
@@ -2908,9 +2931,9 @@ void CreateInertial(TiXmlElement *_elem,
 }
 
 ////////////////////////////////////////////////////////////////////////////////
-void CreateJoint(TiXmlElement *_root,
+void CreateJoint(tinyxml2::XMLElement *_root,
                  urdf::LinkConstSharedPtr _link,
-                 ignition::math::Pose3d &/*_currentTransform*/)
+                 ignition::math::Pose3d &_currentTransform)
```

Comment:
I would revert the change on this line to keep `_currentTransform` commented out since it is generating an unused variable clang warning on the homebrew CI

Reply:
[![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=sdformat-ci-pr_any-homebrew-amd64&build=1312)](https://build.osrfoundation.org/job/sdformat-ci-pr_any-homebrew-amd64/1312/) https://build.osrfoundation.org/job/sdformat-ci-pr_any-homebrew-amd64/1312/warnings10Result/

Reply:
Done